site/src/component/Review/Review.tsx

```diff
@@ -160,8 +131,7 @@ const Review: FC<ReviewProps> = (props) => {
               key={review._id}
               course={props.course}
               professor={props.professor}
-              colors={getU(review._id) as VoteColor}
-              colorUpdater={updateVoteColors}
+              updateScore={(newUserVote) => updateScore(review._id!, newUserVote)}
```

Comment:
Do we need to pass the `updateScore` function? Or could we just define this function within the SubReview since it just uses the redux slice? Also this could maybe be a new reducer on the slice instead.